src/orb.yml.hbs

```diff
@@ -716,7 +725,9 @@ commands:
                 --container-definitions "${CCI_ORB_AWS_ECS_CONTAINER_DEFS}" \
                 "$@" \
                 --output text \
-                --query 'taskDefinition.taskDefinitionArn')
+                --query 'taskDefinition.taskDefinitionArn' \
+                --profile << parameters.profile-name >>)
```

Comment:
Could this be updated such that --profile will not be set if the value of profile-name is ""?